### DISCUSSION TITLE
《大明王朝1566》  | /posts/1566/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#635] [Announcements]
https://yuanfan.rbind.io/posts/1566/

**Comment by hongtaoh** [comment]
你看得真的很仔细。你提到的那些穿帮镜头和最后提到的细节，我从来没有注意过。这部剧给我印象深刻的是严世蕃，感觉演得不错，至少和我读《明朝那些事儿》时脑子里想的差别不大。但是这部剧里的张居正和我想得不太一样。

**Reply by earfanfan** [reply]
莫西莫西，干嘛不给俺展开讲讲，差别不大和不太一样都是撒子哟？

**Reply by hongtaoh** [reply]
主要是看《明朝那些事》时我的印象是张居正是一个非常帅、非常高、非常睿智、非常狠的一个人。电视剧里的他感觉有些软，也没有那么睿智。

**Reply by earfanfan** [reply]
在电视剧里，张居正还没撒势力，徐阶和高拱还在，所以他比较温和，等到将来万历继位，他得势以后，他才会是你心里的那个印象吧。另外，其实电视剧里一直在比较隐晦地表达他的睿智，只不过都掩盖在温和的言行之下了。最后两集，裕王还是裕王，张居正就在跟裕王提出改革的建议，并且直指某些阶层不纳税的问题。

**Reply by earfanfan** [reply]
等等，为何会是非常帅和非常高？

**Reply by hongtaoh** [reply]
明朝那些事里面的描写，就让我有那种感觉。

**Reply by earfanfan** [reply]
嘿嘿嘿，我想吐个槽。有没有可能是你的脑子里把睿智、有能力等人物魅力相关的意象和身高样貌的审美绑定在一起了呢？如果给这个猜想再加段瞎分析的话，是酱紫的：

可能是受电视剧的影响，电视剧往往一演就好多集，主演们的模样常常会被观众们注视许久，能够吸引观众看下去的与演员有关的大约就是两点，演员的样貌好看和演员能散发出极大的个人魅力，后者往往需要很长时间的锻造，相对来说好看的样貌更好找到，所以咱们长大的那个年代电视剧里会有很多好看的样貌，再加上那个年代描绘故事通常都会有界限分明的正派反派之分，电视剧里的戏份很多的正派角色常常被描摹为智慧与美貌并重，于是乎当你被《明朝那些事》里面张居正这个人物所吸引时，不自觉地就会想象出来他拥有符合你审美的外在，也就是非常高非常帅。

**Reply by earfanfan** [reply]
哈，我还有个更没边的吐槽，你小子将来讨老婆一定会找到一个非常高非常美的女孩。

**Reply by hongtaoh** [reply]
可能潜意识里有那样的刻板印象。主要是《明朝那些事儿》里确实也正面说了张居正很帅。至于很高，那确实是我的刻板印象。你后面的猜测可确实没边了。

**Comment by Tlhey** [comment]
好厉害呜呜

**Reply by earfanfan** [reply]
呜呜呜，你也很厉害（ps依依说的）。
